gigpress.php

```diff
@@ -674,6 +673,270 @@ function fetch_gigpress_venues() {
 	return ( $venues !== FALSE) ? $venues : FALSE;
 }
 
+function gigpress_country_list() {
```

Comment:
```suggestion
/**
 * Gets the Gigpress country list.
 *
 * @since TBD
 *
 *
 * @return array $gp_countries The country list.
 */
function gigpress_country_list() {
```